content/en/platform/corda/4.5/enterprise/app-upgrade-notes.md

```diff
@@ -15,7 +15,7 @@ title: Upgrading a CorDapp to a newer platform version
 # Upgrading CorDapps to newer Platform Versions
```

Comment:
In this file there is a link to header ref: 
app-upgrade-notes.html#platform-version-5-gradle-changes
It should be: 
app-upgrade-notes.html#step-2-update-gradle-version-and-associated-dependencies

And this:
app-upgrade-notes.html#cordapp-upgrade-version-numbers-ref
to: 
app-upgrade-notes.html#platform-version-matrix

Reply:
I wasn't able to find those mentions in the app-upgrade-notes.md file, did you mean a different file? Sorry, my brain is working 20% worse on a Friday ^^'

Reply:
1st one appears on line 202. 
2nd one appears on line 345.
Definitely app-upgrade-notes. There is another file called app-upgrade-notes-enterprise, so perhaps you were in that one??